core/lcsb/oidc.py

```diff
@@ -6,22 +6,29 @@
 from django.conf import settings
 from django.db import transaction
 from keycloak import KeycloakAdmin
+from keycloak.exceptions import KeycloakGetError
 
 from core.models.contact_type import ContactType
 from core.models.contact import Contact
 from core.models.partner import Partner
 from core.models.user import User
```

Comment:
those imports are not used. along with 

import json
from multiprocessing import Lock
from django.db import transaction
from core.synchronizers import AccountSynchronizationException,

Reply:
thanks, imports removed